strangerstudios/paid-memberships-pro
# Fix payment settings not saving.

Fixes a minor issue of the get_option call that does not prefix our payment settings.
### All Submissions:


### How to test the changes in this Pull Request:

1.  Before checking out this PR, on V3.0 save settings like API keys inside payment settings for any gateway.
2. Once the page reloads, you should see that the input fields/text fields are empty (but data is still stored inside the DB).
3. Checkout this PR and reload the payment settings page - this will now load values.

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
